### PR TITLE
Helper for grid-related e2e tests

### PR DESCRIPTION
**Pull request by andrzejewsky**
Example usage:

```js
it("Test product click", { tags: ["@productsList"] }, () => {
    // click on cell at column 1 and row 0
    cy.clickGridCell(1, 0)
  },
);

```

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [ ] pages
12. [ ] sales
13. [ ] vouchers
14. [ ] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations
21. [ ] navigation
22. [ ] variants
23. [ ] payments

CONTAINERS=1

### CODE REVIEW

File: cypress/support/e2e.js [hunk 1]
```diff
@@ -48,6 +50,43 @@ Cypress.Commands.add("addAliasToGraphRequest", operationName => {
...
+    It seeks for react fiber node to obtain Glide grid instance.
+    Within its ref, we have getBounds API which returns an info about given cell.
+
+    Browsing over the three starts from canvas. In case of changing that selector,
+    this function must be adjusted as well.
+  */
+  cy
+    .get(SHARED_ELEMENTS.dataGridTable)
+    .should("be.visible")
+    .wait(3000)
```
Comment (andrzejewsky) [inline cypress/support/e2e.js:64]: can we somehow determine if the grid has loaded data instead of adding timeout (i saw it in the other places)?

Comment (wojteknowacki) [inline cypress/support/e2e.js:64]:
```suggestion
ensureCanvasStatic(PRODUCTS_LIST.dataGridTable).then
```

Comment (andrzejewsky) [reply cypress/support/e2e.js:64]: everywhere grid testid stays the same, so it's rather shared element than contexted just for the products

**Review by krzysztofzuraw — APPROVED** [review]

File: cypress/support/e2e.js [hunk 2]
```diff
@@ -18,11 +18,14 @@ import "@percy/cypress";
...
 import { commandTimings } from "cypress-timings";
 
+import { SHARED_ELEMENTS } from "../elements/shared/sharedElements";
 import { urlList } from "../fixtures/urlList";
 import cypressGrep from "../support/cypress-grep/support";
 
 commandTimings();
 
+commandTimings();
+
```
Comment (wojteknowacki) [inline cypress/support/e2e.js:28]: 2x commandTimings() ? 🤔

**Review by wojteknowacki — APPROVED** [review]

**Review by wojteknowacki — APPROVED** [review]

**Review by Droniu — APPROVED** [review]

**Review by krzysztofzuraw — APPROVED** [review]

**Pull request merged**